### DISCUSSION TITLE
Share the Word Of the Program

### DISCUSSION THREAD

**Discussion by mendelsphotography** [ThioJoe/YT-Spammer-Purge#253] [General]
Hi, ThioJoe why don't you get this public Like Collab with other youtubers Eg (Jim Brown)...  about this so more people get to know about it? 

Like what's the Point in the program if you don't get it out there the help other creators

**Comment by Kurdakov** [comment]
it's wrong complain. ThioJoe made videos for utility on his channel, announced it on twitter several times.

as for me - when spammers really annoyed me, I searched 'youtube spammer github', second link was ThioJoe's program with functionality I needed

**Reply by mendelsphotography** [reply]
ok did not realize But At Least maybe someone can try to get it out on a bigger youtube channel like Jim Brown... someone who will speak about it which will bring it to the tons of people's attention

**Reply by mendelsphotography** [reply]
How Many times i have seen videos of youtubers complaining about it But when you reply in the comments it does not help I Mean to try to get the same attention  the Return Youtube Dislike Chrome extension got attention

**Comment by TechStudent10** [comment]
I feel like if ThioJoe wanted to collab, he would do it now after all the major updates that have occured.

**Comment by KendallDoesCoding** [comment]
Well, Idk if Jim Browning will like say yes to this, cause he focusses on tech support scammers, etc. But, a youtuber named [Pleasant Green](https://www.youtube.com/c/PleasantGreen) made a video about youtube scammers, so @ThioJoe can reach out to him https://www.youtube.com/watch?v=-CpkNJx_7bs

**Reply by mendelsphotography** [reply]
well there all pretty much similar i watch them both and that's exactly why i say Jim, Jim browning had his account hacked a few months ago that's Number 1   2 is I am almost postivate he mentioned the issue of youtube spam comments in that video and others and so did many people like them yes actually Pleasant green might be better since he loves to have fun with them Jim Browning has a bigger viewership and is more well known. But i guess it doesn't really matter who as long its someone who does Scambaiting. @KendallDoesCoding

**Reply by KendallDoesCoding** [reply]
Yeah, I too watch Jim, Scammer Payback, Trilogy Media and Pleasant Green. But yeah, I forgot Jim got hacked, but again he'll have to agree to the idea if @ThioJoe is down, if ThioJoe himself isn't then yeah. But definitely, people need to know about this tool. This dude who makes lots of Comment Bot videos who is a small channel got a lot of people hooked up when he made a video about this project - https://www.youtube.com/watch?v=XKNVEGVApTM @mendelsphotography

**Reply by TechStudent10** [reply]
I am subscribed to Jim Browning and Scammer Payback but I don't watch them that often so I'm not sure that YouTube scams are a priority. But that would spread awareness because having multiple YouTubers on board would spread to bigger YouTubers. But if people comment enough, then that's also enough.

**Reply by KendallDoesCoding** [reply]
> I am subscribed to Jim Browning and Scammer Payback but I don't watch them that often so I'm sure that YouTube scams are a priority. But that would spread awareness because having multiple YouTubers on board would spread to bigger YouTubers. But if people comment enough, then that's also enough.

Indeed!

**Reply by Kurdakov** [reply]
probably if mods of https://www.reddit.com/r/youtube/ ( https://www.reddit.com/r/youtube/about/moderators/ just don't spam them, if you already messaged them, let us know, but maybe ThioJoe himself could message them ) would share the word of the utility that will be fine,
there were few comments about this project there under spam related posts - but maybe it's possible to push mods to make something?

**Reply by mendelsphotography** [reply]
For sure idk why I never thought of that

**Comment by miquelfire** [comment]
JayTwoCents made a passing remark about this tool (before the 2.0 release though) because people on his channel were getting scammed.

**Reply by TechStudent10** [reply]
Send the link, please.

**Reply by miquelfire** [reply]
https://www.youtube.com/watch?v=MHLNW0UQwcI

ThioJoe already posted a comment on that video.

**Comment by Kurdakov** [comment]
two tech video bloggers complain about spam attacks
https://twitter.com/ZacksJerryRig/status/1478890194585604099

some people offered them to use YT Spammer Purge, still no action, but maybe they will get to use it and share the word....

btw after observing channels heavily infested with whatsapp spam it so nice to come to clean moderated channel :)

**Comment by mendelsphotography** [comment]
👏👏👏😂Linus Said about it and he said he is planning to make a video about it coming soon. Finally, that's exactly what I meant [https://youtu.be/xMRylDsrxyU?t=505](url) Hopefully, from there it will get known all over and stop the scammers.

**Reply by TechStudent10** [reply]
You also watch WAN show, I see. I was gonna post this, but it appears you have beat me to it.

**Reply by miquelfire** [reply]
Linus posted the video today: https://www.youtube.com/watch?v=zo_uoFI1WXM

**Reply by TechStudent10** [reply]
YES

**Comment by KendallDoesCoding** [comment]
amazing

**Comment by mendelsphotography** [comment]
Well yes FInaly now We got it MKBHD to make a video https://youtu.be/1Cw-vODp-8Y My Whole Goal of this thread was to do that We Should get their Attention of them which will slowly bring more YouTubers to speak about and ultimately over time more and more will fix it without youtube.

**Reply by TechStudent10** [reply]
Let's goo!

**Comment by TechStudent10** [comment]
Chris Titus Tech mentioned this in his latest video.

**Reply by DinhHuy2010** [reply]
video url please

**Reply by TechStudent10** [reply]
https://www.youtube.com/watch?v=OzoDeoSPQ0U
